bulbs/recirc/static/recirc/js/video-recirc.js

```diff
@@ -32,7 +34,7 @@ VideoRecircList.prototype.videoRecircFetched = function(data) {
       'data-track-label' : videoHref,
       'html': $('<figure>',{
         'class' : 'content',
-        'html' : '<div class="image"><img src="' + posterSource + '/16x9/480.jpg"></div>'
+        'html' : '<div class="image"><bulbs-video-play-button></bulbs-video-play-button><img src="' + posterSource + '/16x9/480.jpg"></div>',
```

Comment:
Yeah it's in the bulbs-elements PR: https://github.com/theonion/bulbs-elements/pull/1